src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -175,10 +221,12 @@ public Response findItem(UUID itemId) throws Exception {
 
     return createOk(itemTranslator.translateItem(item));
   }
-
+  
   @Override
-  public Response listItems(String categoryIdsParam, String locationIdsParam, String search, List<String> sort, Long firstResult, Long maxResults) throws Exception {
+  public Response listItems(String categoryIdsParam, String locationIdsParam, String userIds, String search, List<String> sort, Long firstResult, Long maxResults) throws Exception {
 
+    // TODO: userIds
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=18&issues=AWVh4kVmUj60SLv4Ylyo&open=AWVh4kVmUj60SLv4Ylyo)

[ISSUE_KEY]: AWVh4kVmUj60SLv4Ylyo